dmutils/env_helpers.py

```diff
@@ -16,6 +16,7 @@ def get_api_endpoint_from_stage(stage, app='api'):
     dev_ports = {
         'api': 5000,
         'search-api': 5001,
+        'antivirus-api': 5008,
```

Comment:
I think we have tests for these as well - `TestGetAPIEndpointFromStage`

Reply:
Ohhhhright

Reply:
Fixed